Sources/NIOSSH/Child Channels/SSHChannelData.swift

```diff
@@ -12,7 +12,11 @@
 //
 //===----------------------------------------------------------------------===//
 
+#if swift(>=5.6)
+@preconcurrency import NIOCore
```

Comment:
We should make a note somewhere to update this when NIO adds sendable checking.

Reply:
Tracked in https://github.com/apple/swift-nio-ssh/issues/110